Makefile

```diff
@@ -7,15 +7,19 @@ all: welcome
 	@exit 1
 
 doc:
+	export STRIP_FROM_INC_PATH_LIST=$(find . -type d -name include -printf " \"$PWD/%P\"")
 	"$(MAKE)" -BC doc/doxygen
```

Comment:
I think it might be more sensible to move the export logic into the Makefile which is used here: `doc/doxygen/Makefile`. There it could be executed unconditionally. If the use of git is no good, you could probably use `RIOTBASE` variable as a starting point for `find`.